mlos_bench/mlos_bench/config/schemas/optimizers/optimizer-schema.json

```diff
@@ -12,17 +12,14 @@
             "$comment": "config properties common to all optimizer types.",
             "type": "object",
             "properties": {
-                "minimize": {
-                    "description": "The name of the metric to minimize.",
-                    "$comment": "In oneOf spec below require one of 'minimize' or 'maximize'.",
-                    "type": "string",
-                    "default": "score"
-                },
-                "maximize": {
-                    "description": "The name of the metric to maximize.",
-                    "$comment": "In oneOf spec below require one of 'minimize' or 'maximize'.",
```

Comment:
Did you remove the "oneOf" rule being referenced here?

Reply:
good catch! I'll check it now